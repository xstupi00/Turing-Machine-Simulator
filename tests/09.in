S   A a
S a A  
S   A  
S c M R
M b F b
M a S a
M   S  
A   C a
C a S a
C a S c
   a   b
