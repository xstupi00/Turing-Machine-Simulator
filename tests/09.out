S   a   b
A   a   b
Ca  a   b
Sc  a   b
cM  a   b
cS  a   b
cA  a   b
cCa a   b
cSc a   b
ccM a   b
ccS a   b
ccA a   b
ccCaa   b
ccSca   b
cccMa   b
cccSa   b
cccA    b
cccCa   b
cccSc   b
ccccM   b
ccccS   b
ccccA   b
ccccCa  b
ccccSc  b
cccccM  b
cccccS  b
cccccA  b
cccccCa b
cccccSc b
ccccccM b
ccccccS b
ccccccA b
ccccccCab
ccccccScb
cccccccMb
cccccccFb
